### DISCUSSION TITLE
Account vanished! But I have the recovery code, how can I restore?

### DISCUSSION THREAD

**Discussion by Alex-n-Nat** [Authenticator-Extension/Authenticator#862] [General]
So I added an account to the app in Chrome, but for some reason an account vanished! Not sure how and I'm now locked out. HOWEVER when adding the account it generated a recovery key/code, how do I use it?

**Comment by mymindstorm** [comment]
It depends on the account provider, you should contact the account provider's support to find out how to use your recovery codes.
